src/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+}
+
+body {
+  font-size: 11px;
+}
+
+header {
+  padding: 0 50px;
+  height: 60px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+nav ul {
+  list-style: none;
+  display: flex;
+}
+
+nav li {
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+nav li:last-child {
+  margin: 0;
+}
+
+nav a {
+  text-decoration: none;
+  padding: 23px 0;
+}
+
+nav a:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+  position: relative;
+}
+
+.is-active::after {
+  content: "";
+  width: 100%;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  bottom: 0;
+  left: 0;
+}
+
+main {
+  margin: 50px 0 30px;
+}
+
+.catalog {
+  margin: 0 auto;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-between;
+  width: 944px;
+}
+
+.item {
+  width: 200px;
+  height: 405px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 50px;
+  padding: 31px 15px 15px;
+}
+
+.item-img {
+  text-align: center;
+  margin-bottom: 43px;
+}
+
+img[alt="imac"] {
+  width: 158px;
+}
+
+h3 {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+  margin: 0;
+}
+
+p {
+  margin: 5px 0 0;
```

Comment:
It's a bad note. Will be better to write `margin-top: 5px;` if you need only one side.